docs/source/modules/held_suarez.rst

```diff
@@ -0,0 +1,402 @@
+Dry dynamical core (Held--Suarez) Forcing  
+=======================================================================================
+
+Documentation for ``hs_forcing_mod``. 
+
+
+Summary
+-------
+This module is contains the 'physics options' used by Isca when the flag ``idealised_moist_model`` in ``atmosphere_nml`` is set to ``FALSE``. 
```

Comment:
Do we also want to make clear that if would be preferable to do`from isca import DryCodeBase` for this, not just setting the `idealised_moist_model` flag?